setup.py

```diff
@@ -92,6 +91,18 @@
     "Jinja2<3.0,>=2.10.1",
     # unsupported ES version issue
     "elasticsearch>=6.0.0,<7.14",
+    'Flask>=1.0.4,<2.0',
```

Comment:
I would prefer to upgrade to Flask 2 if possible? Otherwise, it will be harder and harder to upgrade in the future. ping @kprzerwa